CMakeLists.txt

```diff
@@ -786,6 +786,14 @@ if(BOUT_BUILD_EXAMPLES)
   add_subdirectory(examples)
 endif()
 
+##################################################
+# Documentation
+
+option(BOUT_BUILD_DOCS "Build the documentation" ON)
+if (BOUT_BUILD_DOCS)
+  add_subdirectory(manual)
```

Comment:
should probably add `EXCLUDE_FROM_ALL`

Reply:
Because the docs are built with `add_custom_target`, they need explicitly adding to `all`, which I've not done. Happy to add `EXCLUDE_FROM_ALL` here too though

Reply:
I hadn't tested it, from the documentation it sounded like without it, that would be added to `all`

Reply:
It does, just for "normal" targets that are created with `add_library` or `add_executable`. `add_custom_target/command` have an `ALL` option to control this separately. I'm not sure why they felt the need to have different behaviour!